tidalapi/media.py

```diff
@@ -620,7 +620,11 @@ def get_file_extension(stream_url: str, stream_codec: Optional[str] = None) -> s
         if AudioExtensions.FLAC in stream_url:
             # If the file extension within the URL is '*.flac', this is simply a FLAC file.
             result: str = AudioExtensions.FLAC
-        elif AudioExtensions.MP4 in stream_url:
```

Comment:
Looks like this shouldn't break any tests but perhaps a good idea to check anyways.

Also, it would be a good idea to extend the existing tests to check that M4A extension does indeed return Codec.MP4A...

Reply:
This is an example, which returns a "m4a" extension: https://tidal.com/browse/track/65119559

Tests are always a good idea. I can do some testing work, as soon as I have finished my personal issues with my project.

Reply:
Added you a test for M4A file extension. To be honest: Someone could also add a lot more tests, but this takes a lot of time. Tell me, if I can help you with something else. Otherwise I would be happy about a merge and hotfix release.